docs/users/quickstart.rst

```diff
@@ -87,8 +91,24 @@ Load data
 
 To load data, just make a right-click in the :ref:`geocatalog` and choose "Add" or "Import".
 
-* Add: link to a file (vector, alphanumeric, ...),
-* Import: import a file and convert it into a table.
+* Add: Link a new database table to a file (vector, alphanumeric, ...),
+	* Pro:
+		* Immediate table creation,
+		* Does not take more disk space,
+		* Can create spatial index.
+	* Cons:
+		* Read-only table,
+		* Cannot create alpha-numeric index,
+		* Only compatible with h2 database.
```

Comment:
Nope we can connect to Postgis table with dblink https://www.postgresql.org/docs/9.2/static/dblink.html